addons/l10n_tr_nilvera_einvoice/wizard/account_move_send.py

```diff
@@ -0,0 +1,168 @@
+from io import BytesIO
+import logging
+
+from odoo import _, api, fields, models, SUPERUSER_ID
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_tr_nilvera_einvoice_enable_xml = fields.Boolean(compute='_compute_l10n_tr_nilvera_einvoice_enable_xml')
+    l10n_tr_nilvera_einvoice_checkbox_xml = fields.Boolean(
+        string="Send E-Invoice to Nilvera",
+        default=True,
+        company_dependent=True,
+    )
+    l10n_tr_nilvera_warnings = fields.Json(compute='_compute_l10n_tr_nilvera_warnings')
+
+    def _get_wizard_values(self):
+        # EXTENDS 'account'
+        values = super()._get_wizard_values()
+        values['l10n_tr_nilvera_einvoice_xml'] = self.l10n_tr_nilvera_einvoice_checkbox_xml
+        return values
+
+    @api.model
+    def _get_wizard_vals_restrict_to(self, only_options):
+        # EXTENDS 'account'
+        values = super()._get_wizard_vals_restrict_to(only_options)
+        return {
+            'l10n_tr_nilvera_einvoice_checkbox_xml': False,
+            **values,
+        }
+
+    @api.model
+    def _get_default_l10n_tr_nilvera_einvoice_enable_einvoice(self, move):
+        return (
```

Comment:
parenthesis not needed

Reply:
I did it to avoid long lines but I don't really mind it if you do, I'll oneline it :)